### PR TITLE
Feature/front end updates

### PR DESCRIPTION
**Pull request by guavajellyaaron**
1. Added a fun plugin for the home page called typed.js. Let me know what you think.
2. Added landing pages for Games, Bosses and Zones.

### CODE REVIEW

File: projects/BossCheatSheets.Web/Controllers/HomeController.cs [hunk 1]
```diff
@@ -38,10 +46,68 @@ public ActionResult AllStrategies()
...
+                    Name = game.Name,
+                    ImagePath = String.Format("{0}.jpg", Helpers.GetCleanURL(game.Name)), 
+                };
+                gamesList.Add(gameVm);
+            }
+            return View(gamesList);
+        }
+
+        [HttpGet]
+        [Route("Bosses/{id?}")]
```
Comment (happygaijin) [inline projects/BossCheatSheets.Web/Controllers/HomeController.cs]: This route makes it look like I am getting a certain boss of id. Not a list of bosses for a game. I think the URL for a list of bosses for a game should be:

/Games/{id}/Bosses

File: projects/BossCheatSheets.Web/Controllers/HomeController.cs [hunk 2]
```diff
@@ -38,10 +46,68 @@ public ActionResult AllStrategies()
...
+                                                                   Helpers.GetCleanURL(boss.Name),
+                                                                   boss.BossID),
+                };
+                bossesList.Add(bossVm);
+            }
+            return View(bossesList);
+        }
+
+        [HttpGet]
+        [Route("Zones/{id?}")]
```
Comment (happygaijin) [inline projects/BossCheatSheets.Web/Controllers/HomeController.cs]: Same idea as above.

/Games/{id}/Zones

File: projects/BossCheatSheets.Web/Views/Shared/_HomeLayout.cshtml [hunk 3]
```diff
@@ -33,6 +34,23 @@
...
         </section>
     </div>
+    <script>
+        $(function () {
+            var bossNames = [];
+            $.get('/Strategy/GetAllBossNames', function (data) {
+                for (var i = 0; i < 3; i++) {
+                    bossNames.push("^1000 " + data[i].name);
+                }
+                bossNames.push("^1000 Something Awesome!");
```
Comment (happygaijin) [inline projects/BossCheatSheets.Web/Views/Shared/_HomeLayout.cshtml]: What is this?

Comment (guavajellyaaron) [inline projects/BossCheatSheets.Web/Views/Shared/_HomeLayout.cshtml]: Here's a quick demo: http://screencast.com/t/fHyCK7Y6BYY

**Comment by happygaijin** [comment]
:+1:

**Pull request merged**